libr/core/cmd_meta.c

```diff
@@ -414,8 +414,20 @@ static int cmd_meta_hsdmf(RCore *core, const char *input) {
 
 	switch (input[1]) {
 	case '?':
-		eprintf ("See C?\n");
-		break;
+		switch (input[0]) {
+		case 'f':
+			eprintf ("Usage: Cf[-] [sz] [fmt..] [@addr]\n\n");
```

Comment:
via r2pipe, eprintf is not catched, so i would use r_cons api instead of eprintf.